play/src/front/Components/AudioManager/AudioManager.svelte

```diff
@@ -70,7 +70,9 @@
             let volume = HTMLAudioPlayer.volume;
             audioPlayerVol.value = "" + volume;
             audioPlayerVolumeIcon.classList.remove("muted");
-            if (volume < 0.3) {
+            if (volume == 0) {
+                onMute();
```

Comment:
If the user defines 0, will be muted. But if he wants to up the volume, it's possible in muted position?

Reply:
Yes it is, it's only to show the Icon of sound in muted mode.